geniza/corpus/management/commands/data_exodus.py

```diff
@@ -0,0 +1,67 @@
+import codecs
+import csv
+
+from django.conf import settings
+from django.contrib.admin.models import ADDITION, LogEntry
+from django.contrib.auth.models import User
+from django.contrib.contenttypes.models import ContentType
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from geniza.corpus.models import Library
+
+
+class Command(BaseCommand):
+    'Import existing data from PGP spreadsheets into the database'
+
+    logentry_message = 'Created via data exodus script'
```

Comment:
this is very minor, but i actually kind of prefer "import" to "exodus" here because it's a little more clear about the data in question being external to django/the database. i can see reserving "exodus" for cases where we can't use a data migration, but still need to move data around in the database for whatever reason (a la cdhweb). but not a strong preference.

Reply:
Fair point! I'm inclined to agree with you, import does seem more clear and obvious.